booster/library/Booster/SMT/Interface.hs

```diff
@@ -156,81 +157,65 @@ translatePrelude def =
 
 checkPrelude :: Log.LoggerMIO io => SMT io ()
 checkPrelude = do
-    runCmd_ $ SetTimeout defaultSMTOptions.timeout
+    -- set large default timeout value for checking the prelude
+    setTimeout defaultSMTOptions.timeout
```

Comment:
The default is `125`. Maybe it isn't that large actually. Either way, I was puzzled by this behaviour because the current code always overrides the user supplied value with `125`, which is what I have done here as well. Perhaps we should just use the user defined value? @geo2a what do you think?

Reply:
Yes, let's use the user defined value here.